com.unity.multiplayer.mlapi/Editor/CodeGen/NetworkBehaviourILPP.cs

```diff
@@ -138,6 +145,21 @@ public override ILPostProcessResult Process(ICompiledAssembly compiledAssembly)
         private MethodReference BitReader_ReadRayPacked_MethodRef;
         private MethodReference BitReader_ReadRay2DPacked_MethodRef;
 
+        private const string NetworkingManager_Singleton = nameof(NetworkingManager.Singleton);
+        private const string NetworkingManager_IsListening = nameof(NetworkingManager.IsListening);
+        private const string NetworkingManager_IsHost = nameof(NetworkingManager.IsHost);
+        private const string NetworkingManager_IsServer = nameof(NetworkingManager.IsServer);
+        private const string NetworkingManager_IsClient = nameof(NetworkingManager.IsClient);
+#pragma warning disable 618
+        private const string NetworkingManager_ntable = nameof(NetworkingManager.__ntable);
+
+        private const string NetworkedBehaviour_BeginSendServerRpc = nameof(NetworkedBehaviour.__beginSendServerRpc);
+        private const string NetworkedBehaviour_EndSendServerRpc = nameof(NetworkedBehaviour.__endSendServerRpc);
+        private const string NetworkedBehaviour_BeginSendClientRpc = nameof(NetworkedBehaviour.__beginSendClientRpc);
+        private const string NetworkedBehaviour_EndSendClientRpc = nameof(NetworkedBehaviour.__endSendClientRpc);
+        private const string NetworkedBehaviour_nexec = nameof(NetworkedBehaviour.__nexec);
+#pragma warning restore 618
+
```

Comment:
could all of the above private const values stand to have a 'k_' in front of them?